src/sdk/clients/decorators/mee/getQuote.ts

```diff
@@ -1041,12 +1053,12 @@ const resolveVerificationGasLimitForPaymentChain = (
       if (index === 0) {
         // return increased verification gas limit for the first userOp
         // as it this userOp will be enabling the permission => requires more gas
-        return { verificationGasLimit: "1000000" }
+        return { verificationGasLimit: 10_00_000n }
```

Comment:
the underscore is in wrong place I believe? unless you wanted to make it 10 Lakh instead of 1M :)